torch/csrc/jit/operator_upgraders/README.md

```diff
@@ -0,0 +1,95 @@
+# Guidance for Operator Developer
+
+After you change either the operator signature in a BC breaking way or the semantics of the operator, you will need to write an “upgrader” to make the change non-BC breaking iff they are used in TorchScript or mobile. In general, you can know your operator is BC breaking, if it fails `test/forward_backward_compatibility/check_forward_backward_compatibility.py `
+
+The steps to write upgrader:
+
+1. Prepare a model before making changes to the operator, following the process:
```

Comment:
Here you mean a test model, right?
Otherwise it's not clear what this model is mean to do or why. Please briefly explain.

Reply:
Add the reason why a test model is needed.